executors/cmdexec/brick.go

```diff
@@ -66,7 +66,7 @@ func (s *CmdExecutor) BrickCreate(host string,
 		fmt.Sprintf("mkfs.xfs -i size=512 -n size=8192 %v", devnode),
 
 		// Fstab
-		fmt.Sprintf("echo \"%v %v xfs rw,inode64,noatime,nouuid 1 2\" | tee -a %v > /dev/null ",
+		fmt.Sprintf("awk \"BEGIN {print \\\"%v %v xfs rw,inode64,noatime,nouuid 1 2\\\" >> \\\"%v\\\"}\"",
```

Comment:
Not sure why awk is really used here.

Why not use echo with a regular `>> %v` instead of that weird `| tee -a %v > /dev/null` ? E.g.:

```
- fmt.Sprintf("echo \"%v %v xfs rw,inode64,noatime,nouuid 1 2\" | tee -a %v > /dev/null ",
+ fmt.Sprintf("echo \"%v %v xfs rw,inode64,noatime,nouuid 1 2\" >> %v",
```
What's the difference of this compared to the awk variant, except for the awk variant being much more cryptic, imho?

Reply:
Oh, I got it! The `>>` in the awk command is not a redirect, but it's basically the open mode (append) in the awk code... That makes sense to me.